jobs/models.py

```diff
@@ -8,36 +8,65 @@ class JobType(models.TextChoices):
     INTERNSHIP = '3', 'Internship'
 
 
+class Location(models.TextChoices):
+    Tel_Aviv = '1', 'Tel Aviv'
+    Jerusalem = '2', 'Jerusalem'
+    Haifa = '3', 'Haifa'
+
+
 # Jobs Model
 class Job(models.Model):
     title = models.CharField(max_length=100)
     description = models.CharField(max_length=255)
-    location = models.CharField(max_length=100)
+    location = models.CharField(
+        max_length=15,
+        choices=Location.choices,
+        default=Location.Tel_Aviv,
+        blank=True, null=True
+    )
     job_type = models.CharField(
         max_length=15,
         choices=JobType.choices,
         default=JobType.PART_TIME,
         blank=True, null=True
     )
     company_name = models.CharField(max_length=255)
-    company_description = models.CharField(max_length=255)
     post_until = models.DateField()
     is_active = models.BooleanField()
+    marked_count = models.IntegerField(default=0)
+    apply_link = models.URLField(unique=True, null=True)
 
     # Function to create a new job
     @classmethod
-    def create_job(cls, title, description, location, job_type, company_name,
-                   company_description, post_until, is_active):
+    def create_job(cls, title, description, location, job_type,
+                   company_name, post_until, is_active, marked_count, apply_link):
         job = cls(title=title, description=description, location=location,
                   job_type=job_type, company_name=company_name,
-                  company_description=company_description, post_until=post_until, is_active=is_active)
+                  post_until=post_until, is_active=is_active, marked_count=marked_count, apply_link=apply_link)
         job.save()
         return job
 
+    def del_job(self):
```

Comment:
Add a test for this function.
You can look for a similar test in users/test.py.

Reply:
The problem is with the self.job.delete(). For some reason it does work for User but not job . Is there any other way to delete an instance self? I commented the exception in line 51